### ISSUE TITLE
Rename default branch

### ISSUE THREAD

**Issue by jasonaowen** [jasonaowen/recurse-faces#78]
The default branch of this repo used the git default name of `master`. That name has some problematic associations, and its only upside is inertia; in particular, I don't believe the name is in keeping with the "No Subtle -isms" [RC Social Rule](https://www.recurse.com/manual#sub-sec-social-rules).

The convention that the community seems to be settling on as a new name for the default branch is `main`.

Rename the default branch from `master` to `main`. Most of the work involved will be with the integrations with external sites, both research and implementation, and so this issue is meant to document and track that work.

See also:
- [Software Freedom Conservancy statement](https://sfconservancy.org/news/2020/jun/23/gitbranchname/)
- [github/renaming](https://github.com/github/renaming)

**Comment by jasonaowen** [comment]
First, for GitHub itself, I need to push a branch with the desired new name, and then I can [change the default branch](https://docs.github.com/en/github/administering-a-repository/setting-the-default-branch).

I'll also take this opportunity to enable the [branch protection rules](https://docs.github.com/articles/defining-the-mergeability-of-pull-requests/) so that the only way to push to `main` is via a pull request.

**Comment by jasonaowen** [comment]
[Heroku supports deployments via git pushes to either `master` or `main`](https://devcenter.heroku.com/articles/git-branches). I'll need to update the documentation in the README.

**Comment by jasonaowen** [comment]
Travis CI might just work; there are no scheduled (cron) builds for this repository, which are set per-branch, and our build is otherwise incredibly simple.

There are also no open PRs that would need to be re-targeted, as called out by [github/renaming](https://github.com/github/renaming). I think it's time to just try it!

**Comment by jasonaowen** [comment]
Travis CI was still showing the default branch as `master` instead of `main`, even after opening and then merging #79. I had to manually trigger a sync from GitHub to Travis by clicking the "Sync account" button at https://travis-ci.org/account/preferences . Once I did so, it detected the new default branch: https://travis-ci.org/github/jasonaowen/recurse-faces/branches .

I think that's everything - GitHub, Heroku, and Travis CI. Glad that was a relatively small change!
